src/main/java/edu/cornell/kfs/fp/businessobject/USBankRecordFieldUtils.java

```diff
@@ -0,0 +1,206 @@
+   */
+  public static KualiDecimal extractDecimal(String line, int begin, int end, int lineCount) throws IOException {
+    KualiDecimal theDecimal;
+    try {
+      String sanitized = line.substring(begin, end);
+      sanitized = StringUtils.remove(sanitized, '-');
+      sanitized = StringUtils.remove(sanitized, '+');
+      theDecimal = new KualiDecimal(sanitized);
+    } catch (StringIndexOutOfBoundsException | IllegalArgumentException e) {
+      // May encounter a StringIndexOutOfBoundsException if the string bounds do not match or 
+      // an IllegalArgumentException if the Decimal does not parse correctly
+      throw new IOException("Unable to parse " +  line.substring(begin, end) + " into a decimal value on line " + lineCount);
+    }
+    return theDecimal;
+  }
+
+  /**
+   * Extracts a KualiDecimal representing dollars and cents from a substring less any ending whitespace 
+   * for a given beginning and ending position.
+   * 
+   * @param line Superstring
+   * @param begin Beginning index
+   * @param end Ending index
+   * @param lineCount The current line number
+   * @return The KualiDecimal parsed from the trimmed substring
+   * @throws IOException When unable to parse the KualiDecimal
+   */
+  public static KualiDecimal extractDecimalWithCents(String line, int begin, int end, int lineCount) throws IOException {
+    KualiDecimal theDecimal;
+    KualiDecimal theCents;
+    try {
+      String sanitized = line.substring(begin, end-2);
+      sanitized = StringUtils.remove(sanitized, '-');
+      sanitized = StringUtils.remove(sanitized, '+');
+      theDecimal = new KualiDecimal(sanitized);
+      theCents = new KualiDecimal(line.substring(end-2, end));
+      theCents = theCents.multiply(new KualiDecimal("0.01"));
+      theDecimal = theDecimal.add(theCents);
+    } catch (StringIndexOutOfBoundsException | IllegalArgumentException e) {
+      // May encounter a StringIndexOutOfBoundsException if the string bounds do not match or 
+      // an IllegalArgumentException if the Decimal or Cents do not parse correctly
+      throw new IOException("Unable to parse " +  line.substring(begin, end) + " into a decimal value on line " + lineCount);
+    }
+    return theDecimal;
+  }
+
+  /**
+   * Extracts a cycle Date from a substring less any ending whitespace for a given beginning and ending position.
+   * 
+   * @param line Superstring
+   * @param begin Beginning index
+   * @param end Ending index
+   * @param lineCount The current line number
+   * @return The cycle Date parsed from the trimmed substring
+   * @throws IOException When unable to parse the cycle Date
+   */
+  public static Date extractCycleDate(String line, int begin, int end, int lineCount) throws IOException {
+    String day;
+    Calendar cal = Calendar.getInstance();
+    Date theDate;
+    try {
+      day = line.substring(begin, end);
+      theDate = KfsDateUtils.newDate(cal.get(Calendar.YEAR), cal.get(Calendar.MONTH), Integer.parseInt(day));
```

Comment:
This was already implemented as-is, I only moved it from the ProcurementCardFlatInputFileType class to this class.